spec/Graph.coffee

```diff
@@ -539,13 +539,13 @@ describe 'Graph', ->
         chai.expect(g.initializers.length).to.equal 0
 
   describe 'with no nodes', ->
-    g = new graph.Graph
+    g = new noflo.graph.Graph
     it 'should not allow adding edges', ->
       g.addEdge 'Foo', 'out', 'Bar', 'in'
-      chai.expect(graph.edges).to.be.empty
+      chai.expect(noflo.graph.edges).to.be.empty
```

Comment:
Like you broke the test here? Or it was broken in first place? Should be fixed anyhow